index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Portfolio page">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+        integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul class="navbar">
+                <li class="logo">
+                    Welcome
+                </li>
+                <li class="menu_icon">
+                    <i class="fa-solid fa-bars"></i>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <div class="header">
+            <h1>Hey there. I'm Agape. <br><span class="heading2">I'm a software developer</span></h1>
+            <p>I can help you build your awesome product, feature or website with the latest trends.
+                Look through my work and experience the magic! If you love what you see then don't hesitate to contact
+                me.
+                Thank you.
+            </p>
+            <div class="icons">
+                <div class="column">
+                    <a href="#"><img src="images/Githubicon.svg" alt="Icon"></a>
+                </div>
+                <div class="column">
+                    <a href="#"><img src="images/Linkedinicon.svg" alt="Icon"></a>
+                </div>
+                <div class="column">
+                    <a href="#"><img src="images/Twittericon.svg" alt="Icon"></a>
+                </div>
+                <div class="column">
+                    <a href="#"><img src="images/Angelisticon.svg" alt="Icon"></a>
+                </div>
+                <div class="column">
+                    <a href="#"><img src="images/Mediumicon.svg" alt="Icon"></a>
+                </div>
+            </div>
+            <div class="dropdown_icon">
+                <img src="images/dropdown_icon.svg" alt="dropdown_icon">
+            </div>
```

Comment:
- To help improve the semantics of your website, I suggest it is better to wrap this in a `section` tag instead of a `div` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.Please fix this 🙏